src/components/IconState.vue

```diff
@@ -1,42 +1,39 @@
 <template>
   <span class="icon-state">
     <v-icon color="primary" @click.stop="dialog = true">{{ localIcon }}</v-icon>
-    <v-dialog v-model="dialog" max-width="400">
+    <v-dialog v-model="dialog" persistent max-width="400">
       <v-card>
         <v-card-title class="headline">Please choose your Icon</v-card-title>
-        <v-card-text>
-          <v-row>
-            <div class='pa-5 mb-5 rounded-t-xl' color="info">
-              Please enter the name of icon you wish to use for this state. You can get the icon
-              names from the following sites.<br/>
-            <v-chip
-              class="mr-2"
-              @click="openMaterial"
-            >
-              <v-icon class='mr-2'>mdi-vector-square</v-icon>
-              Material
-            </v-chip>
-            <v-chip
-              class="mr-2"
-              rounded
-              @click="openMaterialDesign">
-                <v-icon class='mr-2'>mdi-material-design</v-icon>
-                Material desgin
-            </v-chip>
-            </div>
-          </v-row>
-          <v-row>
-            <div class="local-icon">
-              <v-icon>{{localIcon}}</v-icon>
-            </div>
-            <v-text-field v-model="localIcon"></v-text-field>
-          </v-row>
-        </v-card-text>
-        <v-card-actions>
-          <v-spacer></v-spacer>
-          <v-btn color="green darken-1" text @click="close">Nah...</v-btn>
-          <v-btn color="green darken-1" text @click="chooseIcon">Choose</v-btn>
-        </v-card-actions>
+        <v-form ref="iconForm">
+          <v-card-text>
+            <v-row>
+              <div class="pa-5 mb-5 rounded-t-xl" color="info">
+                Please enter the name of icon you wish to use for this state. You can get the icon names from the
+                following sites.<br />
+                <v-chip class="mr-2" @click="openMaterial">
+                  <v-icon class="mr-2">mdi-vector-square</v-icon>
+                  Material
+                </v-chip>
+                <v-chip class="mr-2" rounded @click="openMaterialDesign">
+                  <v-icon class="mr-2">mdi-material-design</v-icon>
+                  Material desgin
+                </v-chip>
+              </div>
+            </v-row>
+            <v-row>
+              <div class="local-icon">
```

Comment:
It may look better to have an example input field above this , labelled as an example field. Or possibly a bigger version so it is obvious at a glance what is happening in this dialog.

Reply:
the concern needs to be dealt later. A ticket will be created.